app/controllers/people_controller.rb

```diff
@@ -226,7 +226,7 @@ def destroy
       MarketplaceService::Listing::Command.delete_listings(@current_user.id)
 
       communities.each { |community_id|
-        PaypalService::API::Api.accounts_api.delete(community_id: @current_community.id, person_id: @current_user.id)
```

Comment:
And missed the fact that this was people controller. So that comment of mine makes no sense.